### ISSUE TITLE
Unable to use as a plugin in nuxtjs

### ISSUE THREAD

**Issue by IndianTinker** [txlabs/vnodes#13]
Hey,
Thanks a lot for the library. I have a small concern about using it in Nuxtjs as a plugin.
I made a plugin like the following : 

```
import Vue from 'vue';
const vnodes = require('vnodes');
const {Screen, Node, Edge, graph} = vnodes;
Vue.use(vnodes)
```

and I get an error as `Could not find a declaration file for module 'vnodes'. `

Could help me understand how the library can use used inside nuxt? 
Thanks :-)

**Comment by AmareshKulkarni** [comment]
Any guidance on this is greatly appreciated.. In my case, I am getting a window not defined error

ReferenceError
window is not defined

`server.js:159096:3
server.js:159184:3
Module../node_modules/vnodes/lib/svg-pan-zoom/uniwheel.js
server.js:27:30
__webpack_require__
server.js:158354:67
Module../node_modules/vnodes/lib/svg-pan-zoom/svg-pan-zoom.js
server.js:27:30
__webpack_require__
server.js:84172:88
Module../node_modules/esbuild-loader/dist/index.js?!./node_modules/vue-loader/lib/index.js?!./node_modules/vnodes/src/components/Screen.vue?vue&type=script&lang=js&
server.js:27:30
__webpack_require__
server.js:159800:174
Module../node_modules/vnodes/src/components/Screen.vue?vue&type=script&lang=js&
server.js:27:30
__webpack_require__
server.js:159765:94
Module../node_modules/vnodes/src/components/Screen.vue`

**Comment by Nikitoring** [comment]
Yes, I have same problem in my app. I get `vnodes` doesn't find, or `window is not defined`

**Comment by Nikitoring** [comment]
I can run in my app like as 
**template:**
```        <client-only>
          <component ref="screen" :is="screen">
            <template v-if="edges">
              <component v-for="edge in graph.edges" :is="edges" :data="edge" :nodes="graph.nodes" :key="edge.id" />
            </template>
            <template v-if="nodes">
              <component v-for="node in graph.nodes" :is="nodes" :data="node" :key="node.id" />
            </template>
          </component>
        </client-only>
```
**script:**
``` 
async mounted() {
    if (process.client) {
      const Vnodes = await import('vnodes');
      this.graph = new Vnodes.default.graph()
      this.screen = Vnodes.default.Screen;
      this.edges = Vnodes.default.Edge;
      this.nodes = Vnodes.default.Node;
    }
  }
```

**Comment by gr1m1um** [comment]
possibly fixed in 1.1.12
